git-filter-repo.mk

```diff
@@ -0,0 +1,45 @@
+ifneq ($(PROCURSUS),1)
+$(error Use the main Makefile)
+endif
+
+SUBPROJECTS             += git-filter-repo
+GIT_FILTER_REPO_VERSION := 2.29.0
+DEB_GIT_FILTER_REPO_V   ?= $(GIT_FILTER_REPO_VERSION)
+
+git-filter-repo-setup: setup
+	-[ ! -f "$(BUILD_SOURCE)/git-filter-repo-$(GIT_FILTER_REPO_VERSION).tar.gz" ] && \
+		wget -q -nc -O$(BUILD_SOURCE)/git-filter-repo-$(GIT_FILTER_REPO_VERSION).tar.gz \
+			https://github.com/newren/git-filter-repo/releases/download/v$(GIT_FILTER_REPO_VERSION)/git-filter-repo-$(GIT_FILTER_REPO_VERSION).tar.xz
```

Comment:
Use ``GITHUB_ARCHIVE`` rather than downloading a release from the Releases tab. No additional files are provided by the release, so it's unnecessary to use them.

Reply:
`GITHUB_ARCHIVE` doesn't support this form of release archive.

Reply:
``GITHUB_ARCHIVE`` does support tags, and like stated before, the specific release provided by the repository provided no additional files, so we can me "our own" archive. 

If you're so confused about this, the following line is what should replace all that mumbo jumbo; the function exists for you to use it...
```make
$(call GITHUB_ARCHIVE,newren,git-filter-repo,$(GIT_FILTER_REPO_VERSION),v$(GIT_FILTER_REPO_VERSION)
```

Reply:
The release tarball that I am downloading contains the manpage, which `GITHUB_ARCHIVE`'s tarball will not have.

Reply:
Manpages can still be made with the [Makefile](https://github.com/newren/git-filter-repo/blob/main/Makefile#L26-L40), so this isn't really a reason to not use the function.

While it requires git to do so, most people have git installed on their system to deal with Procursus in general, so this isn't an excuse either...

Reply:
It requires you to clone the repo, which would take more lines than my wget currently does.